reloader/reload.sh

```diff
@@ -62,5 +69,5 @@ while [ true ]; do
 
     echo "----------------------------------"
     echo ""
-    sleep $SYNC_PERIOD
+    sleep $SYNC_PERIOD & wait $!
```

Comment:
The process now closes quickly without waiting for the docker timeout because of `sleep`